kernel-modules/MODULE_VERSION

```diff
@@ -1 +1 @@
-2.3.0-rc1
+2.3.0-rc2
```

Comment:
There's a conflict here cause we've already released `2.3.0`, when you rebase the PR, please bump this version to `2.4.0-rc1`